Explorer/Assets/DCL/PluginSystem/World/TextShapePlugin.cs

```diff
@@ -5,55 +5,49 @@
 using DCL.PluginSystem.World.Dependencies;
 using DCL.SDKComponents.TextShape.Component;
 using DCL.SDKComponents.TextShape.Fonts;
-using DCL.SDKComponents.TextShape.Renderer;
-using DCL.SDKComponents.TextShape.Renderer.Factory;
 using DCL.SDKComponents.TextShape.System;
-using ECS.ComponentsPooling.Systems;
 using ECS.LifeCycle;
 using System.Collections.Generic;
 using System.Threading;
+using TMPro;
+using UnityEngine;
 
 namespace DCL.PluginSystem.World
 {
     public class TextShapePlugin : IDCLWorldPlugin
     {
-        private readonly ITextShapeRendererFactory textShapeRendererFactory;
         private readonly IPerformanceBudget instantiationFrameTimeBudgetProvider;
         private readonly IComponentPoolsRegistry componentPoolsRegistry;
+        private readonly IFontsStorage fontsStorage;
 
-        public TextShapePlugin(IPerformanceBudget instantiationFrameTimeBudgetProvider, IComponentPoolsRegistry componentPoolsRegistry, IPluginSettingsContainer settingsContainer) : this(
-            instantiationFrameTimeBudgetProvider,
-            componentPoolsRegistry,
-            settingsContainer.GetSettings<FontsSettings>().AsCached()
```

Comment:
Can you please move `FontsSettings` to the settings of **this** plugin just like it's done for everything else?